packer/scripts/infrasim.yml

```diff
@@ -0,0 +1,74 @@
+---
+#This file is used as InfraSIM configuration file
+
+#Node type of your infrasim compute, this will determine the
+#bmc emulation data and bios binary to use.
+#Supported compute node names:
+#	quanta_d51
+#	quanta_t41
+#	dell_c6320
+#	dell_r630
+#	s2600kp
+#	s2600tp
+#	s2600wtt
+type: quanta_d51
+
+compute:
+
+    cpu:
+        #Set node cpu num, the default value is 2
+        #
+        quantities: 2
+
+        #Set node cpu type, the default value is Haswell
+        #Supported type:
+        #	core2duo:  Intel(R) Core(TM)2 Duo CPU     T7700  @ 2.40GHz
+        #	coreduo:   Genuine Intel(R) CPU           T2600  @ 2.16GHz
+        #	Nehalem:   Intel Core i7 9xx (Nehalem Class Core i7)
+        #	Westmere:  Westmere E56xx/L56xx/X56xx (Nehalem-C)
+        #	SandyBridge:  Intel Xeon E312xx (Sandy Bridge)
+        #	IvyBridge:  Intel Xeon E3-12xx v2 (Ivy Bridge)
+        #	Haswell:  Intel Core Processor (Haswell)
+        #	Broadwell:  Intel Core Processor (Broadwell)
+        #   host:  default, KVM processor with all supported host features
+        #          (only available in KVM mode)
+        #
+        type: Haswell
+
+    memory:
+        #Set node memory size, the unit is MB.
+        #The default vaule is 512MB
+        #
+        size: 1024
```

Comment:
Recommend 4GB memory. It will be more promising to install most of operating systems.